packages/aws-rfdk/lib/core/lib/utils.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export class Utils {
+  /**
+   * This method compares two version strings
+   *
+   * @param versionFirst
+   * @param versionSecond
+   *
+   * @returns negative value if first version is smaller than second version;
+   * 0 if the versions matches, positive value if first version is smaller
```

Comment:
There is an error in the description. These 2 statements contradict:

***negative*** value if first version is smaller than second version
***positive***  value if first version is smaller than second version.
```suggestion
   * 0 if the versions matches, positive value if first version is greater
```

Reply:
fixed. good catch.